src/legacy/index.php

```diff
@@ -522,11 +516,11 @@
                                     <label>
                                         Тип прокси-сервера:
                                         <select name="proxy_type" id="proxy_type" class="myinput" title="Тип прокси-сервера">
-                                            <option value="http" <?php echo ($cfg['proxy_type'] == 'http' ? "selected" : "") ?>>HTTP</option>
-                                            <option value="socks4" <?php echo ($cfg['proxy_type'] == 'socks4' ? "selected" : "") ?>>SOCKS4</option>
-                                            <option value="socks4a" <?php echo ($cfg['proxy_type'] == 'socks4a' ? "selected" : "") ?>>SOCKS4A</option>
-                                            <option value="socks5" <?php echo ($cfg['proxy_type'] == 'socks5' ? "selected" : "") ?>>SOCKS5</option>
-                                            <option value="socks5h" <?php echo ($cfg['proxy_type'] == 'socks5h' ? "selected" : "") ?>>SOCKS5H</option>
+                                            <option value="http" <?php echo($cfg['proxy_type'] == 'http' ? "selected" : "") ?>>HTTP</option>
```

Comment:
Do we have a convention to keep quotation marks consistency?

Reply:
No, at least at this moment.